### PR TITLE
Quick fix for the fpscontroller

### PR DESCRIPTION
**Pull request by Jsgoller1**
**TL;DR - I edited a script attached to an imported standard asset, and it isn't saved in git because it's a unity standard asset; I saved the edited version in Assets/Script and attached it to the FPSController**

In order to get wallrunning to function correctly, I had to set the FPSController's gravity variable to near-zero. It's a private variable, so I added a getter and a setter to the script and called them in my wallrunning script. I added the FPSController script to the branch, but because we don't save the standard asset imports (which includes FPSController), it gets overwritten when you re-import (which you will need to do on clone). This PR adds the custom script to Assets/Scripts and then to the FPSController object. Tested and functioning locally.

### CODE REVIEW

**Comment by giniwren** [comment]
runs, looks good.
